eternalcombat-api/src/main/java/com/eternalcode/combat/fight/tagout/TagOutCommand.java

```diff
@@ -0,0 +1,94 @@
+package com.eternalcode.combat.fight.tagout;
+
+import com.eternalcode.combat.config.implementation.PluginConfig;
+import com.eternalcode.combat.notification.NotificationAnnouncer;
+import com.eternalcode.combat.util.DurationUtil;
+import dev.rollczi.litecommands.argument.Arg;
+import dev.rollczi.litecommands.command.execute.Execute;
+import dev.rollczi.litecommands.command.permission.Permission;
+import dev.rollczi.litecommands.command.route.Route;
+import org.bukkit.entity.Player;
+import panda.utilities.text.Formatter;
+
+import java.time.Duration;
+import java.time.Instant;
+import java.util.UUID;
+
+@Route(name = "tag-out", aliases = "tag")
```

Comment:
Recently, there have been huge protests about commands containing -, rename them if you can.

Reply:
this looks awful but ok 

![image](https://github.com/EternalCodeTeam/EternalCombat/assets/80779749/a3c6d316-df18-488b-a302-777bca5ba7e9)